src/System.Private.CoreLib/src/System/Runtime/ProfileOptimization.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Runtime
+{
+    public static class ProfileOptimization
```

Comment:
These stubbed out implementations should be in https://github.com/dotnet/corert/blob/master/src/System.Private.CoreLib/src/System/Runtime/Loader/AssemblyLoadContext.cs instead. Your implementation in CoreFX will work for both CoreCLR and CoreRT that way.

Reply:
Clear